Delcam.ProductInterface.PowerMILL/Classes/PMEntitiesCollection.cs

```diff
@@ -174,6 +174,28 @@ public string GetNewEntityName(string basePrefix = null)
             return "1";
         }
 
+        /// <summary>
+        /// Draws all entities
+        /// </summary>
+        public void DrawAll()
+        {
+            System.Reflection.FieldInfo[] fi = typeof(T).GetFields(System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Static);
+            string Identifier = fi[0].GetValue(null).ToString();
+            if (Identifier != "MODEL")
```

Comment:
What happens if it is a Model?

Reply:
OK, i've done some testings:

**UNDRAW MODEL ALL**
This command works and seems to be the same as
```
VIEW MODEL ; SHADE OFF
VIEW MODEL ; WIREFRAME OFF
```
So we can delete the if condition in the Undraw Function


**DRAW MODEL ALL**
This will cause an Error:
```
Command Error:
DRAW MODEL ALL
Unsupported feature or command
```

Reply:
I think the command for models might just be **DRAW ALL**

Reply:
**DRAW ALL** will draw all entities